packages/inspect-utils/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "@effection/atom": "2.0.3",
-    "@effection/dispatch": "2.0.1",
+    "@effection/dispatch": "2.0.3",
```

Comment:
This is another example of where we need `convector` to support `^` dependencies.

@jbolda Can you point us to roughly where we would need to make the change?